operator/src/main/java/oracle/kubernetes/operator/rest/RestServer.java

```diff
@@ -321,6 +350,13 @@ private SSLContext createSSLContext(KeyManager[] kms) throws Exception {
     return result;
   }
 
+  private static String readCertFromDataOrFile(String data, String file) throws IOException {
```

Comment:
This seems weird - readFromDataOrFile always returns PEM.  either data is pem, or the file is base64 pem.

What's this method supposed to return?  base64 pem? it will receive the same data/file as readFromDataOrFile.
If it's supposed to return base64 pem, then if it uses data, it needs to base64 encode it.